tests/engine/processes/calcjobs/test_calc_job.py

```diff
@@ -262,6 +262,9 @@ def test_portable_code(file_regression, tmp_path, aiida_localhost):
     """test run container code"""
     import pathlib
     (tmp_path / 'bash').write_bytes(b'bash implementation')
+    subdir = tmp_path / 'sub'
+    subdir.mkdir()
+    (subdir / 'dummy').write_bytes(b'dummy')
```

Comment:
Now you still need to check it is actually written to the remote working directory.

Reply:
That's true. Added.